clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTask.groovy

```diff
@@ -57,12 +69,22 @@ class JedisTask implements Task {
     repository.addToHistory(repository.currentState(this).update(TaskState.COMPLETED), this)
   }
 
+  @Deprecated
```

Comment:
would be nice to, in a subsequent PR, to just get rid of these - i don't love how we use `@Deprecated` and never actually deprecate

Reply:
Oh, that's definitely the plan! I actually delete `@Deprecated` 💯💯💯